js/battle.js

```diff
@@ -4761,6 +4761,7 @@ var Battle = (function () {
 					actions += "" + poke.getName() + " fell straight down!";
 					poke.removeVolatile('magnetrise');
 					poke.removeVolatile('telekinesis');
+					if (poke.lastmove === 'fly' || poke.lastmove === 'bounce') poke.sprite.animReset();
```

Comment:
`animReset` is just an animation, not a message, isn't it?

And I still don't understand why we shouldn't `animReset` all the time...